docs/developer-guide/ksqldb-clients/contributing.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: Contributing a new ksqlDB client
+tagline: Contributing a new ksqlDB client
+description: Contribute a new ksqlDB client for your favorite language.
+keywords: ksqlDB, client
+---
+
+This page contains information and recommendations for how to contribute a new [ksqlDB client](index.md).
+
+Functionality
+-------------
+
+Clients should support the following functionality:
```

Comment:
Should or must?

Reply:
Good question. Depends on the resolution to the discussion in https://github.com/confluentinc/ksql/pull/5628/files#r441087931.

Reply:
Based on discussion below, I think we should leave this as "should".